web/inc/main.php

```diff
@@ -4,8 +4,12 @@
 use PHPMailer\PHPMailer\SMTP;
 use PHPMailer\PHPMailer\Exception;
 
-require 'vendor/autoload.php';
+if(!file_exists('vendor/autoload.php')){
+    trigger_error('Unable able to load required libaries. Please run v-add-sys-phpmailer in command line');
+    exit('Unable able to load required libaries. Please run v-add-sys-phpmailer in command line');
```

Comment:
Should exit not only have an exit code?

Reply:
https://www.php.net/manual/en/function.exit.php

As we use it also in CLI might be smart return an exit code